src/EventGrid/EventGrid/ChangeLog.md

```diff
@@ -19,6 +19,21 @@
 -->
 ## Upcoming Release
 
+## Version 1.1.1
```

Comment:
@ahamad-MS you can remove this header and keep all of the text under the `## Upcoming Release` header -- we'll do the versioning for this branch in a separate PR

Reply:
Thx. fixed.

Reply:
@ahamad-MS it doesn't look like this comment was resolved

Reply:
hhhmm.. Weird. re-fixing this.